test/infrastructure/docker/internal/controllers/dockermachine_controller.go

```diff
@@ -27,7 +27,7 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	infrav1 "sigs.k8s.io/cluster-api/test/infrastructure/docker/api/v1beta1"
-	"sigs.k8s.io/cluster-api/test/infrastructure/docker/docker"
+	docker2 "sigs.k8s.io/cluster-api/test/infrastructure/docker/internal/docker"
```

Comment:
```suggestion
	"sigs.k8s.io/cluster-api/test/infrastructure/docker/internal/docker"
```
I assume this was not intentional?

Reply:
fixed